apis/core/v1beta1/featureflag_types.go

```diff
@@ -31,14 +31,19 @@ type FeatureFlagSpec struct {
 }
 
 type FlagSpec struct {
-	Flags map[string]Flag `json:"flags"`
+	Flags `json:",inline"`
```

Comment:
This change was needed due to marshaling of the structure to json. If we want to have the json in the format:

```
{
    "flags": {...}
}
```

The parameter needs to have a separate struct. If we would leave it in the original way, we will be missing the root "flags" in the marshalled json.

This is not a breaking change. It's just a change of the internal representation of the CR. This change is not visible to the user.